test/serializer_test.exs

```diff
@@ -25,6 +25,38 @@ defmodule JSONAPISerializerTest do
     end
   end
 
+  # The `metadata` field may contain arbitrary key-value data, such as user-supplied JSON.
+  defmodule PostWithMetadataView do
+    use JSONAPI.View
+
+    def fields,
+      do: [
+        :text,
+        :body,
+        :full_description,
+        :inserted_at,
+        :metadata,
+        :_private_attribute,
+        :nonstandard__attribute
+      ]
+
+    def meta(data, _conn), do: %{meta_text: "meta_#{data[:text]}"}
```

Comment:
Please add a space between these two functions.  Only the same functions (name/arity) should be grouped without newlines.

Reply:
yep